assertj-guava/src/main/java/org/assertj/guava/api/MultimapAssert.java

```diff
@@ -313,4 +316,74 @@ public final MultimapAssert<K, V> containsAllEntriesOf(Multimap<? extends K, ? e
     throw assertionError(shouldContain(actual, other, entriesNotFoundInActual));
   }
 
+  /**
+   * Verifies that the actual multimap does not contain the given key.
+   * <p>
+   * Examples:
+   * <pre><code class='java'> Multimap&lt;Ring, TolkienCharacter&gt; elvesRingBearers = HashMultimap.create();
+   * elvesRingBearers.put(nenya, galadriel);
+   * elvesRingBearers.put(narya, gandalf);
+   * elvesRingBearers.put(vilya, elrond);
+   *
+   * // assertion will pass
+   * assertThat(elvesRingBearers).doesNotContainKey(oneRing);
+   *
+   * // assertion will fail
+   * assertThat(elvesRingBearers).doesNotContainKey(vilya);</code></pre>
+   *
+   * @param key the given key
+   * @return {@code this} assertions object
+   * @throws AssertionError if the actual map is {@code null}.
+   * @throws AssertionError if the actual map contains the given key.
+   */
+  public MultimapAssert<K, V> doesNotContainKey(K key) {
+    return doesNotContainKeys(key);
+  }
+
+  /**
+   * Verifies that the actual multimap does not contain any of the given keys.
+   * <p>
+   * Examples:
+   * <pre><code class='java'> Multimap&lt;Ring, TolkienCharacter&gt; elvesRingBearers = HashMultimap.create();
+   * elvesRingBearers.put(nenya, galadriel);
+   * elvesRingBearers.put(narya, gandalf);
+   * elvesRingBearers.put(vilya, elrond);
+   *
+   * // assertion will pass
+   * assertThat(elvesRingBearers).doesNotContainKeys(oneRing, someManRing);
+   *
+   * // assertions will fail
+   * assertThat(elvesRingBearers).doesNotContainKeys(vilya, nenya);
+   * assertThat(elvesRingBearers).doesNotContainKeys(vilya, oneRing);</code></pre>
+   *
+   * @param keys the given keys
+   * @return {@code this} assertions object
+   * @throws AssertionError if the actual map is {@code null}.
+   * @throws AssertionError if the actual map contains the given key.
+   * @throws NullPointerException if the varargs of keys is null
+   */
+  public MultimapAssert<K, V> doesNotContainKeys(K... keys) {
+    return doesNotContainKeysForProxy(keys);
+  }
+
+  protected MultimapAssert<K, V> doesNotContainKeysForProxy(K[] keys) {
+    isNotNull();
+    requireNonNull(keys, "The array of keys to look for should not be null");
+    Set<K> foundKeys = findKeys(actual, keys);
+    if (!foundKeys.isEmpty()) {
+      throw assertionError(ShouldNotContainKeys.shouldNotContainKeys(actual, foundKeys));
```

Comment:
Feel free to favor a static import for better readability: 

```suggestion
      throw assertionError(shouldNotContainKeys(actual, foundKeys));
```